src/assets/css/common.css

```diff
@@ -212,6 +212,12 @@ button {
   transition: 0.2s linear;
 }
 
+.header .theme-toggle.active {
+  border: 2px solid white; /* Border for the active button */
+  border-radius: 8px; /* Optional: Add rounded corners */
+  padding: 5px 15px; /* Adjust padding to fit */
+}
```

Comment:
Do you think that those comments are adding something? 

I mean, seeing `color:red` with a comment "This is the color red" adds nothing. 
If something it looks like chat gpt generated code. 

Please do not use comments to explain variable, function , simple css etc...